examples/sentiments/Modelfile

```diff
@@ -0,0 +1,28 @@
+# Modelfile for creating a sentiment analyzer. 
+# Run `ollama create sentiments -f pathtofile` and then `ollama run sentiments` and enter a topic
+
+FROM orca
+TEMPLATE """
```

Comment:
Do we need to use `TEMPLATE` here? This seems like the kind of thing that could go in `SYSTEM` below.

Reply:
i seem to remember the examples wouldn't work without splitting it out from system. plus this is the only example with template. we document but don't show it